### ISSUE TITLE
Build: Rotate info comes up for non-rotating objects

### ISSUE THREAD

**Issue by com1clyf3** [CompletelyFairGames/dwarfcorp#660]
![image](https://user-images.githubusercontent.com/6562812/39151171-704ce498-4712-11e8-99d5-3cbf32dba1b7.png)

Most objects that can be built do not have are that can be affected by rotating them. Th objects that can be rotated are the door, kitchen table, turret, ladder, lamp, bed, and bookshelf. All others should have this rotate info disabled.

cf2d71db

**Comment by mklingen** [comment]
All objects can be rotated. It's just that most objects are sprites whose appearence doesn't change when rotated. Rotating the object can actually be important becase it defines what direction the dwarf will face when using the object.

**Comment by com1clyf3** [comment]
There's no current indication of which side is the dwarf facing side for the objects that do not change when rotated. Maybe add a little info bubble pointing it out or maybe an icon of a dwarf at the correct side?

**Comment by mklingen** [comment]
Yes, that's a bit confusing. There is a white line currently drawn from the center of the object toward the direction the dwarf will try to stand. It doesn't matter that much in practice because the dwarf will just stand somewhere else if the preferred stand location is unreachable. I will mark certain objects as disallowing rotation to make this less ambiguous.

**Comment by com1clyf3** [comment]
Alright. Sounds like a solid design fix.
